packages/amplify-migration/src/app_auth_definition_fetcher.test.ts

```diff
@@ -0,0 +1,211 @@
+import { CognitoIdentityProviderClient, DescribeUserPoolCommand, ListGroupsCommand } from '@aws-sdk/client-cognito-identity-provider';
+import { CognitoIdentityClient, GetIdentityPoolRolesCommand, ListIdentityPoolsCommand } from '@aws-sdk/client-cognito-identity';
+import { CloudFormationClient } from '@aws-sdk/client-cloudformation';
+import { AmplifyClient, ListBackendEnvironmentsCommand } from '@aws-sdk/client-amplify';
+import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3';
+
+import { AmplifyStackParser } from './amplify_stack_parser';
+import { BackendEnvironmentResolver } from './backend_environment_selector';
+import { AppAuthDefinitionFetcher } from './app_auth_definition_fetcher';
+import { BackendDownloader } from './backend_downloader';
+
+const mockUserPoolName = 'mockUserPoolName';
+const mockUserPoolID = 'UserPoolId';
+const mockIdentityPoolName = 'mockIdentityPoolName';
+const mockIdentityPoolId = 'IdentityPoolId';
+const mockAppClientId = 'AppClientID';
```

Comment:
what are we doing about the other client id?

Reply:
I think import auth in gen1 needs only 1 client (web): https://docs.amplify.aws/gen1/javascript/build-a-backend/auth/import-existing-resources/